roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -30,9 +30,10 @@ def capsule_lce_args(action, capsule_id, env, lifecycle)
   STDOUT.puts "There are no external capsules to disassociate."
 else
   external_capsule_ids.split("\n").each do |id|
-    lifecycle_environment = get_info_from_hammer("--csv capsule content lifecycle-environments --id #{id}").split("\n")
-    name = get_info_from_hammer("--csv capsule info --id #{id}", 2).chomp
-    external_capsules << {:id => id, :name => name, :lifecycle_environments => lifecycle_environment}
+    external_capsules << {
```

Comment:
I'm just wondering if it should instead do something like:
```ruby
external_capsules = external_capsule_ids.split("\n").map do |id|
  {
    ...
  }
end

Reply:
Nice idea

I updated the style, please take a look and let me know what you think